python-api/Makefile

```diff
@@ -37,18 +39,18 @@ imgurbash.sh:
 unittests:
 	@rm -f ../tools/*.c.gcov
 	@rm -f ../tools/*.gcda
-	@rm -rf reports
-	@mkdir -p reports/coverage/unittests
-	${PYTEST} --cov gstswitch --pep8 -v -s tests/unittests/
-	#-@mv htmlcov/*.* reports/coverage/unittests
+	${PYTEST} --cov gstswitch --cov-report html -v tests/unittests/ -s
+	mkdir -p reports/coverage/
+	rm -rf reports/coverage/unittests-${PYTHONVERSION}
```

Comment:
Does this need a `-` prefix? I have a feeling that `reports/coverage/unittests-${PYTHONVERSION}` doesn't exist until it's been run once?